apps/simple-trading-app/tsconfig.json

```diff
@@ -10,6 +10,7 @@
     "noImplicitOverride": true,
     "noPropertyAccessFromIndexSignature": false,
     "noImplicitReturns": true,
+    "noImplicitAny": false,
```

Comment:
It's possible to add in a few places some types and we could avoid adding this implicit any

Reply:
Check it please https://github.com/vegaprotocol/frontend-monorepo/pull/970

Reply:
yes, wanted to do that but wasn't sure of some of the types in simple-trading-app and didn't want to break anything.